File: MCPlayerKit/Classes/Demo/List/PlayerCategroryCell.m
//
// Created by majiancheng on 2018/1/3.
// Copyright (c) 2018 majiancheng. All rights reserved.
//

#import "PlayerCategroryCell.h"

#import "Dto.h"
#import "PlayerCategroryDto.h"

@interface PlayerCategroryCell ()

@end


@implementation PlayerCategroryCell {

}

- (instancetype)initWithStyle:(UITableViewCellStyle)style reuseIdentifier:(nullable NSString *)reuseIdentifier {
    self = [super initWithStyle:style reuseIdentifier:reuseIdentifier];
    if (self) {
        self.accessoryType = UITableViewCellAccessoryDisclosureIndicator;
    }
    return self;
}

- (void)loadData:(Dto *)dto {
    [super loadData:dto];
    self.textLabel.text = ((PlayerCategroryDto *) dto).name;
}

@end

File: PlayerKitDemo/Classes/Demo/List/PlayerCategroryCell.swift
//
// Created by majiancheng on 2018/5/28.
// Copyright (c) 2018 majiancheng. All rights reserved.
//

import UIKit

class PlayerCategroryCell: TableViewcell {
    override init(style: UITableViewCellStyle, reuseIdentifier: String?) {
        super.init(style: style, reuseIdentifier: reuseIdentifier)
        self.accessoryType = .disclosureIndicator
    }

    required init?(coder aDecoder: NSCoder) {
        super.init(coder: aDecoder)
    }

    override func loadData(data: Dto) {
        super.loadData(data: data)
        let dto = data as! PlayerCategroryDto
        self.textLabel!.text = dto.name
    }
}